packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -170,7 +171,7 @@
                         x-transition:leave-start="scale-100 opacity-100"
                         x-transition:leave-end="scale-95 opacity-0"
                     @endif
-                    @class([
+                    {{ $attributes->class([
```

Comment:
Instead of `$attributes`, you should use a new bag that only contains the window attributes. Otherwise, you now have leaking of all attributes from the main bag onto the window as well

Maybe pass through the entire `$extraModalWindowAttributeBag` instead of just `$extraModalWindowAttributes`

Reply:
Done. I thought `$attributes` was unique-per-element for some reason.

Another alternative could be passing through the array like before and constructing the bag in the @php block of `index.blade.php`. Might be a bit cleaner?